modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -12,42 +13,40 @@ import org.tessellation.schema.trust.{InternalTrustUpdateBatch, PublicTrust, Tru
 object TrustStorage {
 
   def make[F[_]: Monad: Ref.Make]: F[TrustStorage[F]] =
-    for {
-      trust <- Ref[F].of[Map[PeerId, TrustInfo]](Map.empty)
-    } yield make(trust)
-
-  def make[F[_]: Monad](trust: Ref[F, Map[PeerId, TrustInfo]]): TrustStorage[F] =
```

Comment:
I think it's not a bad idea to have two make methods the way it was. The one with ref as parameter can be useful in tests for example so that the non empty trust map can be injected.

Reply:
Reverted the change.